### PR TITLE
Navigation changes within content elements

### PR DESCRIPTION
**Pull request by fkaminski**
As addressed in slack, a draft regarding changes to content elements representations.

-  New navigation/split of content elements according to the backend
~~- Some settings have already been consistently formatted via tables~~

~~If desired, I would also format the other areas accordingly uniformly ...~~

### CODE REVIEW

**Comment by fkaminski** [comment]
As discussed in the clarifying Slack Huddle with Bjarke, we could do the Navi split. Further changes to the formatting are not necessary.

File: docs/manual/article-management/content-elements/accordion.de.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+title: "Akkordion"
```
Comment (netzarbeiter) [inline docs/manual/article-management/content-elements/accordion.de.md]:
```suggestion
title: "Akkordeon"
```

File: docs/manual/article-management/content-elements/accordion.de.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+description: "Inhaltselemente im Bereich Akkordion"
```
Comment (netzarbeiter) [inline docs/manual/article-management/content-elements/accordion.de.md]:
```suggestion
description: "Inhaltselemente im Bereich Akkordeon"
```

File: docs/manual/article-management/content-elements/accordion.de.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+url: "artikelverwaltung/inhaltselemente/akkordion"
```
Comment (netzarbeiter) [inline docs/manual/article-management/content-elements/accordion.de.md]:
```suggestion
url: "artikelverwaltung/inhaltselemente/akkordeon"
```

File: docs/manual/article-management/content-elements/accordion.de.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+aliases:
+    - /de/artikelverwaltung/inhaltselemente/akkordion/
```
Comment (netzarbeiter) [inline docs/manual/article-management/content-elements/accordion.de.md]:
```suggestion
    - /de/artikelverwaltung/inhaltselemente/akkordeon/
```

**Comment by netzarbeiter** [comment]
Why didn't you just make the changes we discussed?

**Comment by fkaminski** [comment]
Will remove the last two commits with the table format today.

**Comment by fkaminski** [comment]
Removed table format/Back to original format with last commit ...

**Review by netzarbeiter — CHANGES_REQUESTED** [review]
Thank you, Franko

**Review by netzarbeiter — APPROVED** [review]

**Pull request merged**